dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -0,0 +1,24 @@
+module SurveyResultsHelper
+
```

Comment:
@laurelfan I'd like to test this show_survey? function.  What's the best way to deal with so many dependencies (language, request, current_user, SurveyResult, DateTime.now)?  Do I use stub?  Do I pass a bunch of parameters to the function?  Some other technique?  (Thanks heaps!)